dataprofiler/tests/test_dp_logging.py

```diff
@@ -0,0 +1,71 @@
+import unittest
+import logging
+
+from dataprofiler import DataLabeler, dp_logging
+from dataprofiler.labelers.regex_model import RegexModel
+
+
+class TestDPLogging(unittest.TestCase):
+
+    def test_set_verbosity(self):
+        # Ensure that logs are written when verbose
+        dp_logging.set_verbosity(logging.INFO)
+        # Level should be set to INFO
+        self.assertEqual(logging.INFO,
+                         dp_logging.get_logger().getEffectiveLevel())
+
+        # Will write "EPOCH i" updates with statistics
+        with self.assertLogs('DataProfiler.labelers.character_level_cnn_model',
+                             level=logging.INFO) as cm:
+            labeler = DataLabeler(labeler_type='structured', trainable=True)
+            labeler.fit(['this', 'is', 'data'], ['UNKNOWN'] * 3, epochs=7)
+
+        # Check that 3 'EPOCH i' updates written for each epoch
+        for i in range(7):
+            # 3 entries indices corresponding to given epoch
+            idxs = [i * 3, i * 3 + 1, i * 3 + 2]
+            for idx in idxs:
+                log = cm.output[idx]
+                epoch_msg = (f"INFO:DataProfiler.labelers."
+                             f"character_level_cnn_model:\rEPOCH {i}")
+                self.assertEqual(epoch_msg, log[:len(epoch_msg)])
```

Comment:
this seems highly dependent on the code below rather than just validating that text is output at different levels as expected. Also, these seem to check to make sure the logs were made rather than the message is actually sent to the stdout which is what verbosity may or may not be preventing.

Reply:
Made it not code-dependant, need to figure out how to capture stdout in test case, currently having issues mocking.